tidy3d/components/time.py

```diff
@@ -63,7 +63,9 @@ def spectrum(
             Time step to weight FT integral with.
             If array, use to weigh each of the time intervals in ``times``.
         complex_fields : bool
-            Whether time domain fields are complex, e.g., for Bloch boundaries
+            Whether time domain fields are complex, e.g., for Bloch boundaries.
+            Note: this parameter is no longer used, as the DFT definition for complex fields
+            has changed to only use the real part.
```

Comment:
When removing this, let's add a note that the real part of the time signal is used only.